dat/scripts/equipopt/wrapper.lua

```diff
@@ -13,4 +14,5 @@ equipopt.soromid  = require 'equipopt.templates.soromid'
 equipopt.pirate   = require 'equipopt.templates.pirate'
 equipopt.thurion  = require 'equipopt.templates.thurion'
 equipopt.proteron = require 'equipopt.templates.proteron'
+equipopt.scavenger   = require 'equipopt.templates.scavenger'
```

Comment:
I don't think you committed the scavenger template.

Reply:
Shouldn't be needed, artifact from my other modding activity...